### DISCUSSION TITLE
Subtitles broken in Microsoft Edge

### DISCUSSION THREAD

**Discussion by DAC1a** [FrostCo/AdvancedProfanityFilter#488] [Q&A] [answered]
I am trying to use APF on Amazon Prime Video with the Edge browser, however when it is toggled on all subtitles go missing. When toggled off they return. This happens whether I try on a laptop or desktop.

I am running Edge version 110.0.1587.56 (Official build) (64-bit)  -   Win10 v. 21H2 (OS Build 19044.2064) on the desktop - 
and Edge version 110.0.1587.50 (Official build) (64-bit)  -  Win 10 v. 21H2 (OS Build 19044.2486) on the laptop

This does not happen on Chrome. Is there a setting that I have missed? Thank you.

**Comment by richardfrost** [comment, marked as answer]
Hello @DAC1a, I'm sorry for the delay on this. Do you happen to have the option turned on in Edge to hide the captions? This is found in the Audio tab under the heading "Captions/Subtitle". If you have chosen anything aside from the "Show all" they might be hidden on purpose.

![image](https://user-images.githubusercontent.com/1507205/223934259-cd13c235-ab4d-4cae-a0a8-8a4fff7d935f.png)

Also, are you installing the extension through the Chrome Web Store, or the Edge Add-ons store?

**Comment by DAC1a** [comment]
I do appreciate your response. Your suggestion worked. I apologize for not
finding that myself. I am new to this extension and still learning it's
capabilities.
The extension was installed through the Microsoft Edge store.

Thank you again.

**Reply by richardfrost** [reply]
You're welcome @DAC1a ! No worries, there are quite a few options. I'm glad we we're able to figure out what was going on though. Have a great day!
